### DISCUSSION TITLE
Can the Minimizable Dialog be cleaned?

### DISCUSSION THREAD

**Discussion by TerazoBatcaveUser** [twilio-labs/paste#3266] [General] [answered]
So far I'm using the MinimizableDialog features with the use of the useChatLogger() hook, unfortunately, I cannot clean the ChatLog history, if I switch to a different conversation from the Conversation API, the previous messages I pushed still show on the history, do you guys have a "clean history" function?

I'm following this example here: https://paste.twilio.design/blog/2022-12-06-announcing-the-conversations-ui-kit

**Comment by TheSisb** [comment]
Hi @TerazoBatcaveUser,

Good callout. We're working on adding a clear function in the next publish of Paste on NPM. Thanks!

**Reply by TheSisb** [reply]
You can track the PR here: https://github.com/twilio-labs/paste/pull/3271

**Comment by nkrantz** [comment, marked as answer]
Hi @TerazoBatcaveUser - #3271 was merged yesterday. Let us know if you have any questions. Thanks!

**Reply by TheSisb** [reply]
Released as of Paste Core 19.3.0
